test/functional/rpc_psbt.py

```diff
@@ -321,7 +321,7 @@ def run_test(self):
 
         self.log.info("Test invalid fee rate settings")
         for param, value in {("fee_rate", 100000), ("feeRate", 1)}:
-            assert_raises_rpc_error(-4, "Fee exceeds maximum configured by user (e.g. -maxtxfee, maxfeerate)",
+            assert_raises_rpc_error(-4, "Fee exceeds maximum configured by user.",
```

Comment:
In d4dceeafc2292cd050cb10e308d73bc8ef84201a (wallet: test: add new transaction error type):

This seems strictly less informative. I would second @josibake in the suggestion to keep mentioning the `maxtxfee` here. Maybe I’m missing something, could you elaborate why you would push that back to a follow-up?

Reply:
This is fixed, resolving.